### PR TITLE
Add xx:AT-BIO-301 thingies (translations)

### PR DESCRIPTION
**Pull request by jolesh**
**Description:**

added a few xx's for feedback

### CODE REVIEW

File: taxonomies/labels.txt [hunk 1]
```diff
@@ -3115,19 +3115,22 @@ it:AT-BIO-301
...
 fi:AT-BIO-302
+xx:AT-BIO-302
 country:en:Austria
 
 <en:EU Organic
 en:AT-BIO-401
 ca:AT-BIO-401
 fi:AT-BIO-401
 it:AT-BIO-401
+xx:AT-BIO-401
```
Comment (stephanegigandet) [inline taxonomies/labels.txt:3133]: Looks good.

If you add a xx: you can remove the other entries except the canonical en: entry.

<en:EU Organic
en:AT-BIO-401
xx:AT-BIO-401

Comment (jolesh) [reply taxonomies/labels.txt:3133]: Okay I thought there is sth about the Robotoff (and others) why it should be kept.

Comment (jolesh) [reply taxonomies/labels.txt:3133]: @stephanegigandet Just give me a short reply so I know how to proceed.

**Review by stephanegigandet — APPROVED** [review]
Perfect, thank you!

**Pull request merged**